### PR TITLE
Create the topic 'Auth0'.

### PR DESCRIPTION
**Pull request by olegburov**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [x] Curating a new topic or collection

************CURATING A NEW TOPIC************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this topic should be curated:

There are around 400 public repositories in Auth0 organization account. Many SDKs for different languages and platforms, as well as quick starters and samples.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/auth0/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+display_name: Auth0
+github_url: https://github.com/Auth0
+logo: auth0.png
+related: single-sign-on, token-based-authentication, json-web-token, openid-connect, oauth, ws-federation, saml, identity-providers, sdk, enterprise-integration 
+released: 2013
+short_description: Auth0 is the new way to solve Identity. 
+topic: auth0
+url: https://auth0.com
+---
+Auth0 is a global leader in Identity-as-a-Service (IDaaS), provides thousands of enterprise customers with a Universal Identity Platform for their web, mobile, IoT, and internal applications. Its extensible platform seamlessly authenticates and secures more than 1.5B logins per month, making it loved by developers and trusted by global enterprises.
```
Comment (MikeMcQuaid) [inline topics/auth0/index.md]: See https://travis-ci.org/github/explore/builds/365937424?utm_source=github_status&utm_medium=notification

Comment (olegburov) [reply topics/auth0/index.md]: @MikeMcQuaid Thanks, I saw it. To my understanding it doesn't like `0` character in the company name `Auth0`. I can rename it to `AuthZero` or `Authzero` or something else, but that will be another company, not Auth0.

Is there a trick to escape the `Auth0` to pass the validation? For example, extend the list of `["Perl", "Pi"]` with value `Auth`.

Comment (styline) [reply topics/auth0/index.md]: hi logically it s free for your exemple

Comment (MikeMcQuaid) [reply topics/auth0/index.md]: @olegburov You could edit this line: https://github.com/github/explore/blob/1a3a268ac614432e7818c38be6ec38292558032b/test/topics_test.rb#L328

**Comment by MikeMcQuaid** [comment]
Fixing CI failure in https://github.com/github/explore/pull/228.

**Comment by MikeMcQuaid** [comment]
Thanks again so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @olegburov!

**Comment by olegburov** [comment]
@MikeMcQuaid Thank you for working with me on this PR!

**Pull request merged**